src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Physical Cluster Replication
+summary: Follow a tutorial to set up physical replication between a primary and standby cluster.
```

Comment:
I think this summary was pasted from the setup task but may need to be tweaked for the overview.

Reply:
Ah thanks for the catch, this might be something I need to check on other pages..